feedback-module/src/assets/styling_classnames.js

```diff
@@ -6,7 +6,7 @@ export const CHECKBOX_STYLE =
   "text-bold text-primary bg-transparent mobile-lg:font-sans-lg mobile-lg-font-sans-lg font-sans-md mobile-lg:margin-y-2 mobile-lg-margin-y-2 margin-y-1 text-no-uppercase";
 
 export const HEADER_STYLE =
-  "radius-top-lg bg-primary display-flex padding-1 mobile-lg:padding-x-2 mobile-lg:padding-y-105 mobile-lg-padding-x-2 mobile-lg-padding-y-105 font-sans-check";
+  "radius-top-lg bg-primary display-flex padding-top-105 padding-bottom-1 padding-x-1 mobile-lg:padding-x-2 mobile-lg:padding-bottom-105 mobile-lg:padding-top-205 mobile-lg-padding-x-2 mobile-lg-padding-top-2 mobile-lg-padding-bottom-105 font-sans-check";
```

Comment:
note: should probably wait until other PR is merged in and move this to sass

Reply:
yeah definitely just left this here as a placeholder so we know what styles to implement

Reply:
should it be `mobile-lg:padding-top-205` or `mobile-lg:padding-top-2`? I noticed that you had `mobile-lg:padding-top-205` first then `mobile-lg-padding-top-2` later on.

Reply:
mobile-lg:padding-top-2